infra/debian/compiler/docs/onecc.1

```diff
@@ -0,0 +1,156 @@
+.\" Manpage for onecc.
+.\" Contact nnfw@samsung.com to correct errors or typos.
+.TH ONECC "1" "August 2021" "onecc version 1.16.1" "User Commands"
```

Comment:
Adding date and version here will let maintenance revise this in every release.
Are you going to that every release?

Reply:
It is the same rule with other man pages in our project. I want to follow the existing rules.
Can you make another issue for this?

Reply:
When I see the several documents, the Title is acceptable for me.

In this case, when we release new version, we need to update this document too.

https://ubuntu-mate.community/t/how-to-create-your-own-man-pages/7931

Reply:
> Can you make another issue for this?

Um? My question was

> Are you going to that every release?

Reply:
```
man vim
man gcc
```

at least these two don't show versions. or is this file for another interface?

Reply:
This file has same rules with other our man docs. I want to update it at once. So I suggest to make another issue for this.

https://github.com/Samsung/ONE/blob/438c765edee6b2b7a3d22a7879a9c438731e93b1/infra/debian/compiler/docs/one-codegen.1#L1
https://github.com/Samsung/ONE/blob/438c765edee6b2b7a3d22a7879a9c438731e93b1/infra/debian/compiler/docs/one-import-bcq.1#L1
...

For reference,
`man git`
`man cmake`
`man cat`

and so many tools have a version in man doc.

Reply:
I don't understand why you don't answer on my simple question `Are you going to that every release?`
Adding version/date is just a concern for the maintenance.

Adding version/date in man page documents is not my interest and I don't care itself.

I don't know technically about man pages and it's hard to read the document.

Please go on without my approval.

Reply:
> I don't understand why you don't answer on my simple question Are you going to that every release?

I thought I had already answered your question like this:

> In this case, when we release new version, we need to update this document too.

As I know, many man doc describe their version. You can see the version and date information if you scroll all the way to the end.

Thank you for review.